src/Makefile.am

```diff
@@ -412,8 +412,6 @@ squid_SOURCES = \
 	refresh.h \
```

Comment:
> PR description: This is another feature which is very much outdated and unused.

Unless we are absolutely certain that nobody uses this feature, I would relax the above statement to say something like "This feature is outdated. It is probably unused. Modern replacements are available or can be easily built."